test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ViewComponentTests.cs

```diff
@@ -140,6 +140,63 @@ public void ViewComponent_View_WithModelParameter_SetsResultViewWithDefaultViewN
         }
 
         [Fact]
+        public void ViewComponent_View_WithNullModelParameter_SetsResultViewWithDefaultViewNameAndNullModel()
+        {
+            // Arrange
+            var viewComponent = new TestViewComponent();
+            viewComponent.ViewData.Model = new object();
+
+            // Act
+            var actualResult = viewComponent.View(model: null as object);
```

Comment:
@Eilon `View(model: null)` won't compile because the compiler can't figure out `TModel`. `View<object>(model: null)` is however identical. My complaint was just that `View(model: null as object)` isn't something I expect anyone to type.